api/src/messaging/receive.py

```diff
@@ -0,0 +1,97 @@
+from channels.db import database_sync_to_async
+from django.contrib.contenttypes.models import ContentType
+from .models import Message
+
+
+async def receive_message(text_data_json: dict) -> dict:
+    message_body = text_data_json["message_body"]
+    previous_message_id = text_data_json["previous_message_id"]
+    tagged_users = text_data_json["tagged_users"]
+    content_type = text_data_json["content_type"]
+    content_object = None
+    object_id = text_data_json["object_id"]
```

Comment:
Add type hints to each variable
message_body: str
previous_message_id: int
tagged_users: List[int]
content_type: int
content_object: ContentType
object_id: int